### PR TITLE
Add CodeQL action

### PR DESCRIPTION
**Pull request by maciaszczykm**
## Test Plan
CI check.

## Checklist
- [x] If required, I have updated the Plural documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [x] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: .github/workflows/codeql.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: CI
+on:
+  push:
+    branches: [ master ]
+    paths-ignore:
+      - '**/*.md'
+  pull_request:
+    branches: [ master ]
+    paths-ignore:
```
Comment (DavidSpek) [inline .github/workflows/codeql.yml:9]: if this is for the frontend only we should probably scope it to the `www` folder.

Comment (maciaszczykm) [reply .github/workflows/codeql.yml]: Done.

**Comment by maciaszczykm** [comment]
@DavidSpek PTAL.

**Comment by maciaszczykm** [comment]
@floreks PTAL.

**Pull request merged**